src/impress/utils/agentic/__init__.py

```
@@ -0,0 +1 @@
+from .agent import llm_agent, provide_llm_context, PipelineContext 
```

Ruff (0.12.2)

`.agent.llm_agent` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)